### PR TITLE
Update to v1.2.0

### PR DESCRIPTION
**Pull request by biglovisa**
- `photos` field to `projects` 
- `average_price_per_tonne` field to `projects`
- `remaining_mass_g` field to `projects`
- `standard` field to `projects`
- validations on `mass_g` field (has to be greater than 1 and less than 2,000,000,000).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@patch-technology/patch",
-  "version": "1.1.1",
+  "version": "1.2.1",
```
Comment (bspellacy) [inline package.json:3]: Incrementing the minor version should reset, the patch version. E.g. `1.2.0`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.1] - 2020-09-17
```
Comment (bspellacy) [inline CHANGELOG.md:10]: `[1.2.0]`

**Review by bspellacy — CHANGES_REQUESTED** [review]
I'm surprised the diff is so large - what version of prettier are you using?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.0] - 2020-09-17
+
+### Added
+
+- `photos` field to `projects`
+- `average_price_per_tonne` field to `projects`
```
Comment (bspellacy) [inline CHANGELOG.md:15]:
```suggestion
- `average_price_per_tonne_cents_usd` field to `projects`
```

File: src/ApiClient.js [hunk 4]
```diff
@@ -411,7 +411,7 @@ class ApiClient {
   hostSettings() {
     return [
       {
-        url: 'https://api.usepatch.com',
+        url: 'https://{defaultHost}',
```
Comment (bspellacy) [inline src/ApiClient.js:414]: Do you know what the cause of this diff is? Where is `hostSettings()` used?

Comment (biglovisa) [reply src/ApiClient.js:414]: Not exactly sure but it seems like it would be a mistake - `defaultHost` is defined below this line.

**Review by bspellacy — APPROVED** [review]
🚢

**Pull request merged**